pages/ksphere/kubeflow/1.0.1-0.5.0/install/konvoy/index.md

```diff
@@ -12,14 +12,14 @@ enterprise: false
   ```bash
   kubectl kudo init
   ```
-* [Download `kubeflow-1.0.1-0.5.0.tgz` tarball](../../download/).
+* [Download `kubeflow-1.0.1_0.5.0.tgz` tarball](../../download/).
 * Install Kubeflow:
   ```bash
-  kubectl kudo install --namespace kubeflow --create-namespace ./kubeflow-1.0.1-0.5.0.tgz
+  kubectl kudo install --namespace kubeflow --create-namespace ./kubeflow-1.0.1_0.5.0.tgz
   ```
 * If you would like to inject additional annotations to KUDO for Kubeflow's default gateway `kubeflow-ingressgateway`, you can pass in the service annotations as parameters:
   ```bash
-  kubectl kudo install --namespace kubeflow --create-namespace ./kubeflow-1.0.1-0.5.0.tgz -p kubeflowIngressGatewayServiceAnnotations='{"foo": "abc","bar": "xyz"}'
+  kubectl kudo install --namespace kubeflow --create-namespace ./kubeflow-1.0.1_0.5.0.tgz -p kubeflowIngressGatewayServiceAnnotations='{"foo": "abc","bar": "xyz"}'
```

Comment:
Please revert these changes. They do not apply to `0.5.0`.

Reply:
Done